library/general/externalip/template.yaml

```diff
@@ -16,7 +19,9 @@ spec:
           properties:
             allowedIPs:
               type: array
+              description: "An allow-list of external IP addresses."
               items:
+                description: "A single IP address."
```

Comment:
Do we docstring individual items in a list when the list is of a primitive type?

Reply:
+1 I think the description for the array is sufficient.

Reply:
removed